ontology/rml-core.owl

```diff
@@ -91,6 +90,14 @@ foaf:name rdf:type owl:AnnotationProperty .
 #    Object Properties
 #################################################################
 
+###  http://w3id.org/rml/baseIRI
+:baseIRI rdf:type owl:ObjectProperty ;
+       rdfs:domain :TriplesMap ;
+       rdfs:comment "Indicates the base IRI of the RDF graph to be created within a triples map."@en ;
```

Comment:
Within is a bit odd, with is better since it indicates that this triples map is used to generate a graph **_with_**.

```suggestion
       rdfs:comment "Indicates the base IRI of the RDF graph to be created with a triples map."@en ;
```